protected/humhub/modules/marketplace/controllers/PurchaseController.php

```diff
@@ -29,7 +29,7 @@ class PurchaseController extends Controller
     /**
      * @var string
      */
-    public $subLayout = '@admin/views/layouts/module';
+    public $subLayout = '@admin/views/layouts/module-old';
```

Comment:
Still required?

Reply:
Please see where it is used, maybe we should somehow move this into new lauyout:
<img width="1149" alt="purchase" src="https://user-images.githubusercontent.com/6995524/148780854-d9f1983c-d45f-426e-8c59-423ac53433d4.png">

Reply:
@yurabakhtin What do you think of following approach?
![image](https://user-images.githubusercontent.com/4736168/148995990-9de0549a-7ea9-49a4-bae5-737959e066ab.png)

Reply:
@luke- Done in commit https://github.com/humhub/humhub/pull/5476/commits/74ea0a6ea228b7db17f447810ed6e8ea1325cd9b